src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double width;
+    private final double length;
+    private String color;
+
+    public Rectangle(double width, double length, String color) {
+        this.width = width;
+        this.length = length;
+        setColor(color);
```

Comment:
You should use the constructor to directly assign values to the fields instead of calling a setter method. This ensures that the object is fully initialized after the constructor has finished executing.